packages/rawlist/package.json

```diff
@@ -22,6 +26,9 @@
     "@inquirer/core": "^0.0.21-alpha.0",
     "chalk": "^5.0.1"
   },
+  "scripts": {
+    "tsc": "tsc"
+  },
```

Comment:
This is unnecessary. We can use `npx tsc` intead.

Reply:
This is for lerna, it allows to run it easily on all packages